### ISSUE TITLE
More detailed examples please 

### ISSUE THREAD

**Issue by l8gravely** [linode/ansible_linode#138]
### Description

I'd love to see more real world examples in this module, where you don't assume uber-knowledge of ansible, and spell out more of the details, to let people get working faster.  For example, the module linode.cloud.instance_info example should really include the debug statement(s) to print out some basic info of the newly created Linode, without having to go and use the linode-cli or some other method.  

### Example Ansible Config

---
- name: Show details on my-linode
   hosts: localhost
   tasks:
     - name: Get info
        linode.cloud.instance_info:
           label: 'my-linode'
           register: my_linode

   - name: Show info
      debug:
            msg="ID= {{ my_linode.instance.id }}, IP(s)= {{ my_linode.instance.ipv4 }}"

Basically, give some nicer examples / tutorials of actually using the code, not just the API itself.

**Comment by l8gravely** [comment]
Looking at the:

# Minimal example. `LINODE_API_TOKEN` is exposed in environment.
plugin: linode.cloud.instance


I haven't a clue how to use this, and maybe it's against the documentation's ethos, but it would be wonderful to have real world  examples on how to call these scripts, instead of just snippets.   

Especially for how to use the above plugin to find all the linodes with a certain tag, and then pull the private IP out so I can loop over them to build the nodebalancer config.

**Comment by lgarber-akamai** [comment]
Closing this issue since it seems to be resolved. Let me know if it is still relevant.
